### PR TITLE
Deprecate flutter

### PR DESCRIPTION
**Pull request by fabriguespe**
[Preview](https://junk-range-possible-git-deprecateflutter-xmtp-labs.vercel.app/docs/build/get-started/overview)
- Updated [SDK Readme](https://github.com/xmtp/xmtp-flutter/blob/main/README.md)
- Remove references to Dart SDK
- All Dart tabs are now hidden

### CODE REVIEW

**Comment by fabriguespe** [comment]
Hey @scotboyd ! What do you think if we just hide the tabs with CSS for now? IN case we need to get this back up in a month or two.

**Comment by scotboyd** [comment]
I have no problem doing it that way if it works.

**Comment by scotboyd** [comment]
I don't think we should rewrite the history of the change log, we should just remove the link and potentially put text that says "This has since been deprecated."

Also I notice that we haven't updated the change log since December.  Should we have updated it for the Denver releases?

**Comment by fabriguespe** [comment]
You are right @scotboyd thank you. I just added back the changelog entries. Regarding the change log yes hasn't been updated this year. Lets start another PR. Sounds good.

File: docs/faq.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ XMTP doesn't currently support smart contract wallets. XMTP has been implemented
 
 ### Which languages and environments does the XMTP SDK support?
 
-The XMTP SDK is [available for multiple languages](/docs/introduction#xmtp-sdks-and-example-apps#sdks), including JavaScript, Kotlin, Swift, and Dart.
+The XMTP SDK is [available for multiple languages](/docs/introduction#xmtp-sdks-and-example-apps#sdks), including JavaScript, Kotli and Swift.
```
Comment (scotboyd) [inline docs/faq.md:155]:
```suggestion
The XMTP SDK is [available for multiple languages](/docs/introduction#xmtp-sdks-and-example-apps#sdks), including JavaScript, Kotlin and Swift.
```

**Review by scotboyd — CHANGES_REQUESTED** [review]
Took another look and noticed a spelling error.

**Review by scotboyd — APPROVED** [review]

**Review by alexrisch — APPROVED** [review]

**Pull request merged**